CTFd/auth.py

```diff
@@ -5,6 +5,7 @@
 from flask import current_app as app
 from flask import redirect, render_template, request, session, url_for
 from itsdangerous.exc import BadSignature, BadTimeSignature, SignatureExpired
+from werkzeug.exceptions import abort
```

Comment:
Why not import from flask?

Reply:
I think was just a mistake when choosing which function to import in my IDE. I'll switch it to `flask.abort`.